gemini/use-cases/code/analyze_codebase_with_gemini_1_5_pro.ipynb

```diff
@@ -0,0 +1,775 @@
+{
```

Comment:
<p>It uses streaming here to improve user experience because this and the following prompts generate a large amount of text. This helps demonstrate best practices for such scenarios.</p>